src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairHMM.java

```diff
@@ -0,0 +1,287 @@
+package org.broadinstitute.hellbender.utils.pairhmm;
+
+import org.apache.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.genotyper.ReadLikelihoods;
+import org.broadinstitute.hellbender.utils.genotyper.LikelihoodMatrix;
+import org.broadinstitute.hellbender.utils.haplotype.Haplotype;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class VectorLoglessPairHMM extends LoglessPairHMM {
+
+    protected final static Logger logger = Logger.getLogger(VectorLoglessPairHMM.class);
+    protected long threadLocalSetupTimeDiff = 0;
+    protected static long pairHMMSetupTime = 0;
+    protected final static Boolean runningOnMac = System.getProperty("os.name").toLowerCase().startsWith("mac");
```

Comment:
Is `os.name` guaranteed to be set on all systems? Might need to guard against null here.